test/es6/ES6NewTarget.js

```diff
@@ -49,7 +49,7 @@ var tests = [
             assert.areEqual('something', obj.target, "The name 'target' can be used as an identifier");
         }
     },
-/*
+
     // Blocked by 'ReferenceError: '<_lexicalNewTargetSymbol>' is undefined' bug

```

Comment:
Done. Thanks!

---
In reply to: [73446652](https://github.com/Microsoft/ChakraCore/pull/1377#discussion_r73446652) [](ancestors = 73446652)